tests/e2e-utils/index.js

```diff
@@ -0,0 +1,44 @@
+import { CustomerFlow, StoreOwnerFlow } from './flows';
```

Comment:
The 3 `.js` files need to be moved to `/tests/e2e/utils/src` to be transpiled by the build script. The package.json needs to be updated to reflect the transpiled files. I'll add a separate comment there.